.github/workflows/ci.yaml

```diff
@@ -16,6 +17,7 @@ jobs:
         os:
           - macos-12
           - macos-13
+          - macos-14
```

Comment:
OS14 is still compatible with some intel based macs. I'm assuming that it's optimized for apple silicon. In testing the binary that's produced, does this fix the lag that was being reported for m-series macs?

Reply:
It halved the time.

Reply:
Woohoo! Great work Qiao!